api/swagger.yaml

```diff
@@ -5238,7 +5238,7 @@ paths:
           schema:
             $ref: "#/definitions/ErrorResponse"
       tags: ["Image"]
-  /deltas/create:
+  /image/delta:
```

Comment:
Shouldn't this be `/images/delta`? Either that or `client/image_delta.go:17` is correct, can't be both :P